sparkle/src/components/Input.tsx

```diff
@@ -18,12 +25,28 @@ const INPUT_STATES = ["error", "disabled", "default"];
 
 type InputStateType = (typeof INPUT_STATES)[number];
 
+const messageVariantStyles: Record<MessageStatus, string> = {
+  info: "s-text-muted-foreground",
+  default: "s-text-muted-foreground",
+  error: "s-text-foreground-warning",
+};
+
 const stateVariantStyles: Record<InputStateType, string> = {
   default: "focus-visible:s-ring-ring",
   disabled:
     "disabled:s-cursor-not-allowed disabled:s-opacity-50 disabled:s-text-muted-foreground",
   error: "s-border-border-warning focus:s-ring-ring-warning",
 };
+
+const messageVariant = cva("", {
+  variants: {
+    status: messageVariantStyles,
+  },
+  defaultVariants: {
+    status: "info",
```

Comment:
Should be aligned with the default value.

Reply:
My bad